src/Directory.Build.props

```diff
@@ -72,11 +72,19 @@
     Force the compilation using Roslyn Generators on net461 (Unit Tests)
     -->
 	  <UnoUIUseRoslynSourceGenerators Condition="'$(TargetFramework)'=='net461'">true</UnoUIUseRoslynSourceGenerators>
-	  
-    <!--
+
+	  <!--
+    Force the compilation using Roslyn Generators on monoandroid
+    -->
+	  <UnoUIUseRoslynSourceGenerators Condition="'$(TargetFramework)'=='monoandroid11.0' or '$(TargetFramework)'=='monoandroid12.0' or '$(TargetFramework)'=='xamarinmac20'">true</UnoUIUseRoslynSourceGenerators>
```

Comment:
`or '$(TargetFramework)'=='xamarinmac20'`
^ that does not match the above comment

Reply:
Indeed, I'm mostly trying out options until the property can unconditionally be set to true 🙂